pkg/migrations/op_create_table.go

```diff
@@ -15,10 +15,17 @@ import (
 var _ Operation = (*OpCreateTable)(nil)
 
 func (o *OpCreateTable) Start(ctx context.Context, conn *sql.DB, stateSchema string, tr SQLTransformer, s *schema.Schema, cbs ...CallbackFn) (*schema.Table, error) {
+	// Generate SQL for the columns in the table
+	columnsSQL, err := columnsToSQL(o.Columns, tr)
```

Comment:
[nit] I believe everywhere else we use `colSQL` naming and here we are using `columnsSQL`

Reply:
I think it's fine because those occurences are for a column (singular) in a helper function; here it's a plural in a 'top level' `Start` function.